Tools/MGCB/BuildContent.cs

```diff
@@ -15,10 +16,20 @@ namespace MGCB
 {
     class BuildContent
     {
+        public bool LaunchDebugger;
+
         [CommandLineParameter(
             Name = "launchdebugger",
             Description = "Wait for debugger to attach before building content.")]
-        public bool LaunchDebugger = false;
+        public void OnLaunchDebugger(bool val)
+        {
+            LaunchDebugger = val;
+
+            if (LaunchDebugger)
+            {
+                Debugger.Launch();
+            }
+        }
```

Comment:
Please pull this out into another PR.  Checking to see if a big PR works and doesn't break things is hard enough without packing in a bunch of separate features.